demisto_sdk/commands/pre_commit/hooks/hook.py

```diff
@@ -1,10 +1,13 @@
+import os
 import re
 from copy import deepcopy
 from pathlib import Path
 from typing import Any, Dict, Iterable, List, Set
 
 from demisto_sdk.commands.common.logger import logger
 
+PROPERTIES_TO_DELETE = {"needs"}
```

Comment:
whats this?

Reply:
Makes sure that added properties (which are not supported by pre-commit) deleted in the hook.